.env.example

```diff
@@ -1,10 +1,14 @@
 # GitHub
+# Generate this with `openssl rand -hex 64` then add it to the GitHub app
 GITHUB_WEBHOOK_SECRET="abc123"
 GITHUB_CLIENT_SECRET="abc123"
 GITHUB_APP_ID="123456"
-GITHUB_PRIVATE_KEY="-----BEGIN RSA PRIVATE KEY-----
+
+# Generate this on the GitHub App dashboard, then convert from PKCS#1 to PKCS#8 following this:
+# https://github.com/gr2m/universal-github-app-jwt#readme
+GITHUB_PRIVATE_KEY="-----BEGIN PRIVATE KEY-----
 abcdef=
------END RSA PRIVATE KEY-----"
+-----END PRIVATE KEY-----"
```

Comment:
@tedspare @sarimrmalik please make note of this change which introduces a new step because I updated the github package. TLDR when you get a .pem file from GH, you need to run `openssl pkcs8 -topk8 -inform PEM -outform PEM -nocrypt -in private-key.pem -out private-key-pkcs8.key` and then use that as the new PK

Reply:
Great catch, 🦅 👁️